jerry-core/api/jerry.c

```diff
@@ -323,7 +323,6 @@ jerry_parse (const jerry_char_t *source_p, /**< script source */
   if (JERRY_CONTEXT (jerry_init_flags) & JERRY_INIT_MEM_STATS_SEPARATE)
   {
     jmem_stats_print ();
-    jmem_stats_reset_peak ();
```

Comment:
what compatibility is that we'd like to keep? is that semantic, API level, or command line level (or something else)? because I see several ways to deprecate this flag. e.g.,
- if we want API compatibility, we could keep the flag in the enum but mark it as deprecated in its doc and don't use it anywhere in jerry-core (perhaps only for logging a warning).
- if we don't want to keep command line compatibility, we could even remove it from the CLI of the jerry tool.